angular/core/models/group_model.coffee

```diff
@@ -169,3 +178,10 @@ angular.module('loomioApp').factory 'GroupModel', (DraftableModel, AppConfig) ->
 
     isSubgroupOfSecretParent: ->
       @isSubgroup() && @parent().privacyIsSecret()
+
+    setupComplete: ->
```

Comment:
Either is ok with me. I've been curious about decorating JS models for a little bit now, but haven't had the occasion to get it to work yet.